pwnlib/util/proc.py

```diff
@@ -333,20 +333,23 @@ def wait_for_debugger(pid, debugger_pid=None):
     t = Timeout()
     with t.countdown(timeout=15):
         with log.waitfor('Waiting for debugger') as l:
-            if debugger_pid:
+            while debugger_pid:
```

Comment:
Does this logic make sense? How is this `while` loop ever meant to to terminate?

Testing this, pwntools hangs on "Waiting for debugger".